source/common/res/features/import-notification/main.js

```diff
@@ -1,11 +1,12 @@
 (function poll() {
-  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.actOnChangeInit === true && typeof ynab.utilities.TransactionImportUtilities !== 'undefined') {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.actOnChangeInit === true && typeof ynab.managers.TransactionImporter !== 'undefined') {
     ynabToolKit.importNotification = function () {
       $('.import-notification').remove();
       $('.nav-account-row').each(function (index, row) {
         var account = ynabToolKit.shared.getEmberView($(row).attr('id')).get('data');
```

Comment:
I do this a lot in my features -- I think I'm going to make a helper to `getElementData` or something that hides all the `getEmberView` stuff when I have time.

no action item for you just putting this in history so I'm bound to do it :3 lol